src/detail-page/components/charts/Chart.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC } from 'react';
+import styled from 'styled-components';
+import { Typography } from '@material-ui/core';
+import { ChartSource } from '../../../data/charts/chartInterfaces';
+import { ProfessionId } from '../../../data/home-page/ITProfessions';
+
+interface ChartInterface {
```

Comment:
It's a small thing, but I'd go with `Props` or `ChildProps` to keep consistency with the rest of the app.

Reply:
You're right. I mixed convention from another project :sweat_smile: